src/Symfony/Component/Console/Application.php

```diff
@@ -159,6 +159,8 @@ public function run(InputInterface $input = null, OutputInterface $output = null
      * @param OutputInterface $output An Output instance
      *
      * @return int 0 if everything went fine, or an error code
+     *
+     * @throws \Exception propagate the exception from `doRunCommand`
```

Comment:
This can be removed as it does not bring any useful information (\Exception can be anything, we only document exceptions when we give an explanation about when a specific exception can occur).